resources/istio/files/job-script.sh

```diff
@@ -45,8 +45,14 @@ if [ ! -z "$overrides" ]; then
   done <<< "$overrides"
 fi
 
-printf "istioctl manifest apply -f /etc/istio/config.yaml ${overrides_transformed}\n"
-istioctl manifest apply -f /etc/istio/config.yaml ${overrides_transformed}
+istio_control_plane_overrides=""
+
+if [ -f "/etc/istio/overrides.yaml" ]; then
+  istio_control_plane_overrides="-f /etc/istio/overrides.yaml"
+fi
+
+printf "istioctl manifest apply -f /etc/istio/config.yaml ${istio_control_plane_overrides} ${overrides_transformed}\n"
+istioctl manifest apply -f /etc/istio/config.yaml ${istio_control_plane_overrides} ${overrides_transformed}
```

Comment:
To consider : if istio control plane is provided perhaps it  'old' overrides aka `overrides_transformed` should be ignored. So if user provides old helm options they are taken into account but if CP is provided then it takes precedence.  Why? there's a chance that merge of values will not properly be applied.